bucket/plex-desktop.json

```diff
@@ -0,0 +1,33 @@
+{
+    "version": "1.30.1.2115-81e1fc3f",
+    "description": "Plex for Windows",
+    "homepage": "https://www.plex.tv/apps-devices/#modal-devices-plex-for-windows",
+    "license": "Proprietary",
+    "architecture": {
+        "64bit": {
+            "url": "https://downloads.plex.tv/plex-desktop/1.30.1.2115-81e1fc3f/windows/Plex-1.30.1.2115-81e1fc3f-x86_64.exe#/dl.7z",
+            "hash": "c9eb3ecb157d32251abaaae3a7b6b8aca1d51f3846ee29f1dcfe2b10ab96b11c"
```

Comment:
```suggestion
            "url": "https://downloads.plex.tv/plex-desktop/1.30.1.2115-81e1fc3f/windows/Plex-1.30.1.2115-81e1fc3f-x86_64.exe#/dl.7z",
            "hash": "sha1:9fd30e82fe8dad74f58edf1f6ba9f1688f4e408a"
```

Reply:
Could you say what is the criteria to use sha1 instead of sha2?

Reply:
The provided hashes are sha1..

Reply:
Ah, I got it.
`checkver`'s response is a sha1